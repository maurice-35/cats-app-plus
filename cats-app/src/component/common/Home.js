import React from 'react'

const Home = () => {

  return (
    <section>
      <div className="container">
        <h1 className="name">
          <span className="logo-emoji">🐈</span>
          <h1>Your Lovely Cats</h1>
          <span className="logo-emoji">🐈</span>
        </h1>
      </div>
    </section>
  )
}

export default Home