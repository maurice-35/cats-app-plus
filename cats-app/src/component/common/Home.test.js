import React from "react";
import { render, screen } from "@testing-library/react"
import Home from "./Home";

test("Home page should be rendered", () => {
	render(<Home />);
	const paragraph = screen.getByText(/Welcome To Your Lovely Cats App/i);
	expect(paragraph).toMatchSnapshot();
});