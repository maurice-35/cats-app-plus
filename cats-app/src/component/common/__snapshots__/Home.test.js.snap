// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page should be rendered 1`] = `
<p
  class="welcome"
>
   Welcome To Your Lovely Cats App
</p>
`;
